src/platform/plugins/shared/visualizations/public/utils/saved_visualize_utils.ts

```diff
@@ -359,20 +359,27 @@ export async function saveVisualization(
   }
 
   try {
-    await checkForDuplicateTitle(
-      savedObject,
-      copyOnSave,
-      isTitleDuplicateConfirmed,
-      onTitleDuplicate,
-      services
-    );
-    const createOpt = {
-      migrationVersion: savedObject.migrationVersion,
-      references: extractedRefs.references,
-    };
```

Comment:
These options are never typed so old props are never culled. `migrationVersion` is no longer supported.